### PR TITLE
Updating cluster computer slide presenter notes.

### PR DESCRIPTION
**Pull request by lcoghill**
Updating presenter notes for cluster computer that were duplicated from the "cloud computing" slides. Attempting to address issue [419](https://github.com/carpentries-incubator/hpc-intro/issues/419).

### CODE REVIEW

File: files/jargon.html
Comment (reid-a) [inline files/jargon.html:149]: At the end of line 149 here, I think "The will" should be "They will"?

**Review by reid-a — CHANGES_REQUESTED** [review]
Grammar error on line 149 -- hopefully an easy fix!

**Comment by reid-a** [comment]
I have made a separate PR, #423, which addresses the CI build error -- once that's merged into the main repo, and incorporated into this PR, things should work.

**Comment by reid-a** [comment]
I have taken the liberty of updating the branch upstream to incorporate the spelling correction -- once the requested changes have been made, this PR should be good to merge, assuming the CI runs successfully.

**Comment by reid-a** [comment]
I should say, once the requested changes have been *addressed* -- you don't have to do what was asked, but you do need to either make the case for the existing version, or propose other changes. Thanks again for contributing!

File: files/jargon.html
Comment (bkmgit) [reply files/jargon.html:149]: Ok

**Comment by reid-a** [comment]
I have taken the liberty of making my requested grammar fix directly on the branch. Will merge ASAP.

Should not be taken to detract from the value of this contribution, this is terrific!

**Review by reid-a — APPROVED** [review]

**Pull request merged**
